build.sbt

```diff
@@ -178,14 +179,19 @@ lazy val db = project
     """.stripMargin.trim
   )
 
-lazy val json = project
+lazy val json = crossProject(JVMPlatform, JSPlatform)
+  .crossType(CrossType.Full)
```

Comment:
I think this project doesn't need to be full. It can be `CrossType.Pure` as there is no js specific code. We can change this in another PR